release_calendar.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: default
+redirect_from:
+- /cal
+- /release-calendar
+title: Release Calendar
+body_class: release-calendar blog_typography post
+max_page_width: medium
+toc: false
+---
+
+<div class="wrap pagetitle">
+  <h1>Release Calendar</h1>
+</div>
+
+DuckLake follows [semantic versioning](https://semver.org/spec/v2.0.0.html).
+Larger new features are introduced in minor versions,
+while patch versions mostly contain bugfixes.
+
+DuckLake 1.0 is planned to be released in 2026.
+
+## Upcoming Releases
+
+Upcoming releases are shown in the table bellow. **Please note that these dates are tentative** and DuckLake maintainers may decide to push back release dates to ensure the stability and quality of releases. DuckLake spec releases are currently tied to DuckLake extension releases, this may change in the future. It is also worth mentioning that some DuckLake extension releases may have a dependency on DuckDB and therefore will need to adjust to the [DuckDB release calendar](https://duckdb.org/release_calendar.html).
```

Comment:
```suggestion
Upcoming releases are shown in the table below. **Please note that these dates are tentative** and DuckLake maintainers may decide to push back release dates to ensure the stability and quality of releases. DuckLake spec releases are currently tied to DuckLake extension releases, this may change in the future. It is also worth mentioning that some DuckLake extension releases may have a dependency on DuckDB and therefore will need to adjust to the [DuckDB release calendar](https://duckdb.org/release_calendar).
```